olp-cpp-sdk-core/include/olp/core/math/AlignedBox.h

```diff
@@ -332,7 +346,7 @@ void AlignedBox<T, N>::Reset(const VectorType& min, const VectorType& max) {
 template <typename T, unsigned int N>
 typename AlignedBox<T, N>::VectorType AlignedBox<T, N>::Center() const {
 #ifndef NDEBUG
-  // Check for possible 64-bit integer overflow/underflow
+  // Checks for possible 64-bit integer overflows or underflows.
```

Comment:
I don't think you should also adapt normal comments as this would be an endless job.